src/Enghouse/readme.md

```diff
@@ -0,0 +1,89 @@
+
+
+The integration utilises the flexible Custom Chat integration to EI **CCaaS**, the steps for deployment are:
+
+
+1. Configure Chat routing in EI **CCaaS** tenant with desired Queue and Skills.
+2. Configure CustomChat "widget" and copy its URL used to control the Live Agent interaction specific to your needs, and a "WA" URL to access the WA integration from EI **CCaaS**.
+3. When using the integration, the "widget" URL is required to be passed either as the *ccURL* parameter to the "WA" URL, or as a variable in the WA Dialog called *CCURL*.  The "WA" URL must also include parameters to utilise your Assistant's *integrationID*, *region*, and *serviceInstanceID*.
+
+
+
+## Example
+
+If the CustomChat widget" URL was:
+
+`https://{Enghouse CCaaS FQDN}/SocialConnectorHelperAPI/api/customchat/getWidgetConfiguration?WidgetId=<*widgetID*>`
+
+...you would set this to the variable *CCURL* in the Dialog, prior to any likelihood of needing to speak to a Live Agent.
+
+
+If the Assistant URL provided to you by Enghouse Interactive was:
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat`
+
+...you would access your Assistant dialog pass your Assistants' parameters to it (using default EI **CCaaS** Watson page in this example):
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat/indexWatson.html?integrationID=<*integration-ID*>&region=<*region*>&serviceInstanceID=<*service-ID*>`
+
+...or if you preferred not to change an existing Dialog, the "widget" URL may be added as a URL-encoded parameter instead of being set in the dialog:
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat/indexWatson.html?integrationID=<*integration-ID*>&region=<*region*>&serviceInstanceID=<*service-ID*>&ccURL=<*widget-URL*>`
+
+
+## URL options
+
+The complete list of parameters that may be passed to the Assistant URL are:
+
+- *integrationID* - Watson Assistant Integration ID
+- *region* - Watson Assistant region
+- *serviceInstanceID* - Watson Assistant Instance ID
+- *subscriptionID* - Watson Assistant Subscription ID
+- *ccName* - the display name of the contact centre, displayed prior to agent answering chat (default is "Contact Centre")
+- *ccURL* - the URL-encoded "widget" URL. This may also be set (unencoded) via the Dialog as the *CCURL* variable (overrides URL if both set). 
+- *agentIcon* - the URL-encoded URL of the agent icon (default is an Enghouse Interactive swirl logo).  This may also be set (unencoded) in the Dialog as the *AgentIcon* variable (overrides URL if both set).
+
+## Demo
+![IBMWatson](https://user-images.githubusercontent.com/30227436/203360082-bd2d1968-0d35-47e3-8039-0acd852bbf67.gif)
+
+## Known limitations and issues
+
+The Watson `watsonAssistantChatOptions` may be created in a custom HTML but both the `onLoad` function and `serviceDeskFactory` are hard-coded by our integration, preventing some customization options in `onLoad`.  If such customization is required please consult with the Enghouse CCaaS team to discuss best solution for your requirements.
```

Comment:
As noted in my other comments I don't think this will be acceptable to a lot of our customers. I will show you an alternative for how to deal with this.